pkg/meta/version_service_test.go

```diff
@@ -479,10 +479,10 @@ func Test_DeleteBucketMeta(t *testing.T) {
 func TestDestination_Parse(t *testing.T) {
 	strPtr := func(s string) *string { return &s }
 	tests := []struct {
+		wantBucket  *string
```

Comment:
is it possible to ignore tests? 
i'd prefer readability to performance in this case.